wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -110,7 +107,7 @@ getHistoryLimited
     -> Maybe Word
     -> m ([CTx], Word)
 getHistoryLimited mCWalId mAccId mAddrId mSkip mLimit =
-    first applySkipLimit <$> getHistory mCWalId mAccId mAddrId
+    first applySkipLimit <$> getHistory mCWalId mAccId mAddrId (skip + limit)
```

Comment:
I don't like you even process transactions which you will skip, i.e. you will try to find addresses of transactions which you will skip in future. It's double work and it might reduce performance

Reply:
maybe it's enough for now, but I suggest to fix it in future

Reply:
It's really non-trivial to do. Pending transactions and transactions from mempool are displayed at the top, but before displaying them we have to exclude from them ones which are from blocks, so I have to fetch db txs at first.
Looks like what I'm doing currently is incorrect due to this

Reply:
I realized, that I said strange thing, nvm